mk/install.mk

```diff
@@ -65,7 +65,7 @@ CFG_ADB_DEVICE_STATUS=$(1)
 endef
 
 $(foreach target,$(CFG_TARGET), \
-  $(if $(findstring $(target),"arm-linux-androideabi"), \
+  $(if $(or $(findstring $(target),"arm-linux-androideabi"),$(findstring $(target),"aarch64-linux-android")), \
```

Comment:
Out of curiosity, it seems a shame to have to duplicate this logic of "check both triples", so would it be possible to just find the string "android" in the target and trigger this logic in that case? Are the android triples where we don't interact with adb?